src/TropicalGeometry/matrix.jl

```diff
@@ -74,3 +74,70 @@ function is_tropically_generic(A::MatrixElem{<:TropicalSemiringElem})
     return helper(transpose(A),nca,nra)
   end
 end
+
+@doc raw"""
+    is_polytrope(A::MatrixElem{<:TropicalSemiringElem})
+
+Check if the tropical convex hull of the rows of `A` is ordinarily convex (ie a polytrope)
+
+# Examples
+```jldoctest
+julia> A = tropical_semiring()[0 0 1; 0 1 0; 0 3 3]
+[(0)   (0)   (1)]
+[(0)   (1)   (0)]
+[(0)   (3)   (3)]
+
+julia> is_polytrope(A)
+true
+```
+"""
+function is_polytrope(A::MatrixElem{<:TropicalSemiringElem})
+  #=Convert the matrix of tropical numbers into matrix of rational numbers in
+  to pass it into polymake's tropical convex hull function =#
+  ncA = ncols(A)
+  nrA = nrows(A)
+  QQA = zero_matrix(QQ,nrA,ncA)
+  for i in 1:nrA
+    for j in 1:ncA
+      iszero(A[i,j]) && return false
+      QQA[i,j] += QQ(A[i,j])
+    end
+  end
+  #Compute the tropical convex hull 
+  tempP = Polymake.tropical.Polytope{convention(A)}(POINTS=QQA)
+  #=Compute the tropical convex hull again, this time with 
+  the minimal generating set of vertices of tropical polytope=#
+  P = Polymake.tropical.Polytope{convention(A)}(POINTS=tempP.VERTICES)
+  PMCV = P.POLYTOPE_MAXIMAL_COVECTORS
+  Polymake.Shell.CV = PMCV
+  #Get the length out of the polymake shell
+  Polymake.shell_execute(raw"""$tmp = $CV->size;""")
+  l = Polymake.Shell.tmp
+  return l == 1
```

Comment:
```suggestion
  return length(P.POLYTOPE_MAXIMAL_COVECTORS) == 1
```
If you change the required Polymake version in the Project.toml to 0.11.20 you can avoid the (slightly fragile) polymake shell commands.

I added `Array{IncidenceMatrix}` as a wrapped type:
```julia
julia> c = Polymake.tropical.cyclic{min}(3,4);

julia> length(c.POLYTOPE_MAXIMAL_COVECTORS)
9

julia> c.POLYTOPE_MAXIMAL_COVECTORS[1]
4×4 IncidenceMatrix
[2, 3, 4]
[1]
[1]
[1]
```